tools/db_crashtest.py

```diff
@@ -44,6 +44,7 @@
     "checksum_type" : lambda: random.choice(["kCRC32c", "kxxHash", "kxxHash64"]),
     "compression_max_dict_bytes": lambda: 16384 * random.randint(0, 1),
     "compression_zstd_max_train_bytes": lambda: 65536 * random.randint(0, 1),
+    "compression_parallel_threads": lambda: random.randint(1, 4),
```

Comment:
Can you give more weight to 1? We want to heavily cover non-parallel case for now.